[esx]/esx_menu_default/client/main.lua

```diff
@@ -20,6 +22,14 @@ local function closeMenu(namespace, name)
     })
 end
 
+AddEventHandler('onResourceStop', function(resource)
+    if GetCurrentResourceName() == resource and OpenedMenus > 0  then
+        ESX.UI.Menu.CloseAll()
+    elseif CurrentNameSpace ~= nil and CurrentNameSpace == resource and OpenedMenus > 0 then
```

Comment:
If CurrentNameSpace does not equal resource it will also automatically mean that it is not nil, so the first check can be removed.

Reply:
Most resources use **GetCurrentResourceName()** for name space.
I will check this error handler with this [native](https://docs.fivem.net/natives/?_0x4D52FE5B)

Reply:
I test native that's return me es_extended.